File: proxy/tests/script_engine/version_set.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local version_set = {
	request = function(flow)
		flow.request.version = Version.HTTP3
	end,
	response = function(flow)
		flow.response.version = Version.HTTP3
	end,
}
Extensions = { version_set }

File: proxy/tests/script_engine/version_set.js
/// <reference path="../../script_libs/js/index.d.ts" />
/** @type {Extension} */
const version_set = {
  request(flow) {
    flow.request.version = Version.HTTP3_0
  },
  response(flow) {
    flow.response.version = Version.HTTP3_0
  }
};
globalThis.extensions = [version_set];
